modules/apps/social/social-bookmarks-taglib/src/main/resources/META-INF/resources/bookmarks/SocialBookmarksDropdownPropsTransformer.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+const SHARE_WINDOW_HEIGHT = 436;
+const SHARE_WINDOW_WIDTH = 626;
+
+function openSocialBookmark({className, classPK, type, postURL, url}) {
+	const shareWindowFeatures = `
+		left=${window.innerWidth / 2 - SHARE_WINDOW_WIDTH / 2},
+		height=${SHARE_WINDOW_HEIGHT},
+		toolbar=0,
+		top=${window.innerHeight / 2 - SHARE_WINDOW_HEIGHT / 2},
+		status=0,
+		width=${SHARE_WINDOW_WIDTH},
+	`;
+
+	window.open(postURL, null, shareWindowFeatures).focus();
+
+	Liferay.fire('socialBookmarks:share', {
+		className: className,
+		classPK: classPK,
+		type: type,
+		url: url,
+	});
+
+	return false;
+}
+
+export default function propsTransformer({items, ...otherProps}) {
+	return {
+		...otherProps,
+		items: items.map(({data, ...otherProps}) => {
```

Comment:
For sure this shouldn't be `...otherProps`...

Reply:
This would be easily changed, so I'll call it `rest` for now for a lack of a better word

Reply:
![good-call](https://user-images.githubusercontent.com/7074/86362805-2455ff80-bc76-11ea-90ca-25329f1f36d2.gif)